### PR TITLE
Update `Disable Lockdown` section to indicate error may from snap

### PR DESCRIPTION
**Pull request by haosdent**
Update `Disable Lockdown` section to indicate error may be from the snap installation way.

##### Checklist

- [x] Language changes are updated in `man/adoc/bpftrace.adoc` and if needed in `docs/reference_guide.md`
- [x] User-visible and non-trivial changes updated in `CHANGELOG.md`
- [x] The new behaviour is covered by tests

### CODE REVIEW

File: INSTALL.md [hunk 1]
```diff
@@ -411,6 +411,10 @@ To test that the build works, you can try running the test suite, and a one-line
 
 # Disable Lockdown
 
+Noted that you may encounter kernel lockdown error if you install bpftrace
```
Comment (viktormalik) [inline INSTALL.md:414]:
```suggestion
Note that you may encounter kernel lockdown error if you install bpftrace
```

File: INSTALL.md [hunk 2]
```diff
@@ -411,6 +411,10 @@ To test that the build works, you can try running the test suite, and a one-line
...
+via `snap` incorrectly. Please refer [Ubuntu](#ubuntu-packages) for more
```
Comment (viktormalik) [inline INSTALL.md:415]:
```suggestion
via `snap` incorrectly. Please refer to [Ubuntu](#ubuntu-packages) for more
```

**Review by viktormalik — COMMENTED** [review]
As this is just a note, I'd suggest moving it to the end of the section.

**Comment by haosdent** [comment]
Thx @viktormalik !

File: INSTALL.md [hunk 3]
```diff
@@ -411,6 +411,10 @@ To test that the build works, you can try running the test suite, and a one-line
 
 # Disable Lockdown
 
+Note that you may encounter kernel lockdown error if you install bpftrace
+via `snap` incorrectly. Please refer to [Ubuntu](#ubuntu-packages) for more
+details regrading how to use `snap` to install `bpftrace`.
+
```
Comment (viktormalik) [inline INSTALL.md:417]: Please move the text to the end of the "Disable Lockdown" section, it will make more sense there.

Comment (haosdent) [reply INSTALL.md]: done

**Review by viktormalik — APPROVED** [review]
Thanks! Could you please squash the commits into one?

**Comment by haosdent** [comment]
@viktormalik done, thank you!

**Pull request merged**